packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -591,11 +591,23 @@ const defaultConfigs = {
     expression: "XAUUSD * USDPERL"
   },
   XAUUSD: {
-    type: "tradermade",
-    pair: "XAUUSD",
-    minuteLookback: 7200,
-    hourlyLookback: 259200,
-    minTimeBetweenUpdates: 60
+    type: "fallback",
+    orderedFeeds: [
+      {
+        type: "tradermade",
+        pair: "XAUUSD",
+        minuteLookback: 7200,
+        hourlyLookback: 259200,
+        minTimeBetweenUpdates: 60
+      },
+      {
+        type: "quandl",
+        // https://www.quandl.com/data/CHRIS/CME_MGC1-E-micro-Gold-Futures-Continuous-Contract-1-MGC1-Front-Month
+        datasetCode: "CHRIS",
```

Comment:
This is the "e-micro gold futures", but there are a few other "gold" related datasets. This one seemed like the best free option but curious what others think

Reply:
This is a futures contract that is expected to be highly correlated with the gold price, so I think it's fine to use, but is it scaled such that it lines up with XAUUSD?

Reply:
It passes the eyeball test that's all  I can say.